web/src/views/InvestigationDetail.vue

```diff
@@ -464,17 +473,78 @@ export default defineComponent({
       }));
     }
 
+    async function toggleNonTiledImageOverlay(pin: Pin, dataset: Dataset) {
+      if (!dataset.content) return;
+      if (!featureLayer) return; // sanity check. this should exist
+      if (!nonTiledOverlayFeature) {
+        nonTiledOverlayFeature = featureLayer.createFeature('quad');
+      }
+      const quadData = nonTiledOverlayFeature.data() || [];
+      // eslint-disable-next-line
+      const existingOverlay = quadData.find((overlay: any) => overlay.pinId === pin.id);
+      if (existingOverlay) {
+        nonTiledOverlayFeature.data(
+          // eslint-disable-next-line
+          nonTiledOverlayFeature.data().filter((overlay: any) => overlay.pinId !== pin.id),
+        ).draw();
+        nonTiledOverlayFeature.draw();
+      } else {
+        try {
+          const urlRoot = process.env.VUE_APP_API_ROOT;
+          const url = `${urlRoot}/datasets/${dataset.id}/tiles/0/0/0.png`;
```

Comment:
Hm. I'd like to think about this one a bit and perhaps consult @manthey.

I'm wondering if I should add an endpoint in `django-large-image` that will pass the whole image.

Otherwise, I would recommend using a thumbnail here rather than the 0-level tile:

```suggestion
          const url = `${urlRoot}/datasets/tile_source/${dataset.id}/thumbnail.png`;
```

(but that change would only be valid AFTER #135 )

Reply:
In the Girder 3 instance, thumbnail can specify a maximum size.  I don't recall if django-large-image supports this.

Reply:
Just added that in django-large-image v0.4.4
![Screen Shot 2022-04-20 at 11 52 47 AM](https://user-images.githubusercontent.com/22067021/164292467-fbbc68bc-d168-43c0-bf3a-4531ccbea898.png)

Reply:
(please ignore the typo in `max_width`'s docs

Reply:
I've applied this change in 7e068b5